components/proxy/src/main/java/com/hotels/styx/admin/handlers/UrlPatternRouter.java

```diff
@@ -116,11 +121,12 @@ public UrlPatternRouter build() {
         private final Pattern pattern;
         private final HttpHandler handler;
         private final List<String> placeholderNames;
+        private static final Pattern PLACEHOLDER_PATTERN = Pattern.compile(":([a-zA-Z0-9-_]+)");
 
         public RouteDescriptor(HttpMethod method, String pattern, HttpHandler handler) {
```

Comment:
For all the methods that declare this 'pattern' parameter (we are calling it 'regex' in some invocations as well), shouldn't we call it  "path", "pathExpression", "pathPattern" or something similar instead?

Reply:
I'm calling it `uriPattern` like they do in Play framework. WDYT?